### PR TITLE
Fix different multipath related issues (CP-19247 and CA-234893)

### PR DESCRIPTION
**Pull request by stefanopanella**
This pr should supercede ps 336 since it is fixing some problems found with pr 336

Revert "CA-208229: enable mpathcount filtering by SCSIid"
CP-19247: Fix mpathcount problems affecting scalability test
CA-234893: sm-multipath.service goes inactive after SM RPM upgrade

### CODE REVIEW

File: etc/mpcount.service [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Unit]
```
Comment (germanop) [inline etc/mpcount.service]: If my previous comment about etc gets accepted, rename this one

File: etc/mpcount.service [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+Description=Multipath Count Service
+
+[Service]
+Unit=forking
```
Comment (germanop) [inline etc/mpcount.service]: Not an expert of systemd at all but should not this be Type=forking?

File: scripts/udevmpath.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo /opt/xensource/sm/mpathcount.py | at now 2> /dev/null
+exit 0
```
Comment (germanop) [inline scripts/udevmpath.sh]: Given I already asked other changes.. missing newline here

File: Makefile [hunk 4]
```diff
@@ -157,6 +157,8 @@ install: precheck
 	  $(SM_STAGING)/$(SYSTEMD_SERVICE_DIR)
 	install -m 644 etc/make-dummy-sr.service \
 	  $(SM_STAGING)/$(SYSTEMD_SERVICE_DIR)
+	install -m 644 etc/mpcount.service \
```
Comment (stefanopanella) [inline Makefile]: OK, can you please suggest how should I change the Makefile to do that?

File: etc/mpcount.service (hunk 1 above)
Comment (stefanopanella) [inline etc/mpcount.service]: as before, where should I put the file? just under multipath dir instead of etc?

File: etc/mpcount.service (hunk 2 above)
Comment (stefanopanella) [inline etc/mpcount.service]: Yes, good catch, I do not know how that ended up like this

File: scripts/udevmpath.sh (hunk 3 above)
Comment (stefanopanella) [inline scripts/udevmpath.sh]: ok I will add a new line

**Pull request closed without merging**